packages/flame_test/lib/src/test_golden.dart

```diff
@@ -29,7 +29,11 @@ void testGolden(
   PrepareGameFunction testBody, {
   required String goldenFile,
   FlameGame? game,
+  bool skip = false,
 }) {
+  if (skip) {
+    return;
+  }
   testWidgets(testName, (tester) async {
```

Comment:
You could just pass in skip here instead

Reply:
ok